website/docs/language/modules/sources.html.md

```diff
@@ -237,16 +237,18 @@ only SSH key authentication is supported, and
 
 By default, Terraform will clone and use the default branch (referenced by
 `HEAD`) in the selected repository. You can override this using the
-`ref` argument:
+`ref` argument. The value of the `ref` argument can be any reference that would be accepted
+by the `git checkout` command, such as branch, SHA-1 hash (short or full), or tag names. The [Git documentation](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection#_single_revisions) contains a complete list:
```

Comment:
```suggestion
by the `git checkout` command, such as branch, SHA-1 hash (short or full), or tag names. The [Git documentation](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection#_single_revisions) contains a complete list.
```

Reply:
no problem at all :D

Reply:
ooo - adding the comment also. only saw the code diff - 1 second

Reply:
there you go - lmk if the comment phrasing is ok